pkg/basicstation/cups/server_test.go

```diff
@@ -138,17 +138,17 @@ func (m *mockGatewayClient) UpdateAPIKey(ctx context.Context, in *ttnpb.UpdateGa
 }
 
 const updateInfoRequest = `{
-  "router": "58a0:cbff:fe80:19",
-  "cupsUri": "https://thethingsnetwork.org:443",
-  "tcUri": "",
-  "cupsCredCrc": 1398343300,
-  "tcCredCrc": 0,
-  "station": "2.0.0(minihub/debug) 2018-12-06 09:30:35",
-  "model": "minihub",
-  "package": "2.0.0",
-  "keys": [
-    392840017
-  ]
+	"router": "58a0:cbff:fe80:19",
+	"cupsUri": "https://thethingsnetwork.org:443",
+	"tcUri": "",
+	"cupsCredCrc": 1398343300,
+	"tcCredCrc": 0,
+	"station": "2.0.0(minihub/debug) 2018-12-06 09:30:35",
+	"model": "minihub",
+	"package": "2.0.0",
+	"keys": [
+		392840017
+	]
```

Comment:
This doesn't look quite right. We shouldn't be indenting JSON with tabs just because it happens to be in a Go file.